### PR TITLE
Resolved browser console errors

### LINKED ISSUE #3239: Remove browser console errors
**Actual Behaviour**

There are lots of console errors in browsers based on css properties of `styled-components` and many more. 

**Expected Behaviour**

Remove the console errors

**Screenshots of the issue**

csvdv

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #3239 

Changes: 
- Renamed css properties such as `backgroundColor` , `borderRadius` etc. to `backgroundcolor` , `borderradius` to avoid conflict of `styled-components`styles with react styles.
  - Link to the issue: https://github.com/mui-org/material-ui/issues/13957

- Used `onCancelSearch` function prop to be used when `<CloseIcon/>` is clicked in `material-ui-search-bar`.

- Added `key` to the list in `skillloader.js`

Demo Link: https://pr-3248-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/ChatApp/MessageComposer.react.js [hunk 1]
```diff
@@ -396,7 +396,7 @@ class MessageComposer extends Component {
           />
         )}
         <TextAreaContainer
-          backgroundColor={textarea}
+          backgroundcolor={textarea}
```
Comment (akshatnitd) [inline src/components/ChatApp/MessageComposer.react.js]: For resolving these warning, using `$` before the variable name is the convention. So, name it `$backgroundColor`

Comment (Dilshaad21) [reply src/components/ChatApp/MessageComposer.react.js]: Ok :+1:

**Comment by Dilshaad21** [comment]
@akshatnitd done!

File: src/components/cms/BrowseSkill/skillLoader.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ const addLoader = () => {
   const limit = isMobileView() ? 2 : 4;
   for (let i = 0; i < limit; i++) {
     loader.push(
-      <ContentContainer>
+      <ContentContainer key={i}>
```
Comment (anishagg17) [inline src/components/cms/BrowseSkill/skillLoader.js]: Duplicate of #3211
Please close that pr as it is a very small enhancement

**Comment by Dilshaad21** [comment]
@akshatnitd I think you closed this PR accidentally after approving my changes

**Pull request merged**